### ISSUE TITLE
App Data?

### ISSUE THREAD

**Issue by Shadeclan** [Aefyr/SAI#183]
Since it appears that Titanium Backup has been abandoned (at least I've been trying for months to get a response out of them without success) and SAI is so much better at backing up and restoring APKs than Titanium Backup (I always had issues even if the APK _wasn't_ split), I was wondering if there were any plans on adding data backup capabilities to SAI?  It would be extremely helpful!

FYI: Performance of SAI on the Motorola G4 with /e/OS installed was flawless - I lost everything on my phone through a careless mistake on my part and SAI allowed me to restore EVERY app - with Titanium Backup, it was always safer to restore the app via the app store first, then restore the data.  Thank you very much!

**Comment by blacklightpy** [comment]
@Shadeclan From README.md:
> # State of SAI
> SAI is kinda dead cause I have no motivation for continuing its development. I will probably maybe still occasionally fix bugs and add small features/improvements, focusing on the installer functionality, but idk. If you're looking for an actual backup solution, you can try [OAndBackupX](https://f-droid.org/packages/com.machiav3lli.backup/) or [Swift Backup](https://play.google.com/store/apps/details?id=org.swiftapps.swiftbackup).

**Comment by Shadeclan** [comment]
Thank you for getting back to me.  OAndBackupX isn't being maintained either and Swift Backup only works if you have the Google Playstore installed, which I don't.  I've had success with the Migrate app which is also not being maintained anymore but it is showing its age.  Last time I switched phones, I had to reinstall half the apps.  Darken, the developer of SD Maid has hinted that he may create an app called Backup Butler but that was nearly 2 years ago.
There's definitely a market out there for decent, unGoogled backup software but nobody's writing anything.
